src/components/tabs/tabs.scss

```diff
@@ -286,3 +286,30 @@ md-toolbar + md-tabs {
   border-top-left-radius: 0;
   border-top-right-radius: 0;
 }
+
+md-tabs[md-rtl] {
```

Comment:
I don't think we should introduce a new attribute just for RTL support.

We normally apply stylings dynamically, by detecting the current language direction.

Check the `rtl-prop`, `rtl-value` mixin.